common/scala/build.gradle

```diff
@@ -40,6 +40,10 @@ dependencies {
     compile "com.typesafe.akka:akka-actor_${gradle.scala.depVersion}:${gradle.akka.version}"
     compile "com.typesafe.akka:akka-stream_${gradle.scala.depVersion}:${gradle.akka.version}"
     compile "com.typesafe.akka:akka-slf4j_${gradle.scala.depVersion}:${gradle.akka.version}"
+    compile "com.typesafe.akka:akka-cluster_${gradle.scala.depVersion}:${gradle.akka.version}"
```

Comment:
One thing to clarify here is, in our(downstream) production system, we are still sticking to `akka-2.5.26` and netty-based transport for akka-remote.
So there can be some differences.